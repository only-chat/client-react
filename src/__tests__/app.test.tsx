import '@testing-library/jest-dom'
import React from 'react'
import { render, screen } from '@testing-library/react'
import App from '../app'

test('renders App', () => {
  process.env.WS_HOST = 'http://github.com/only-chat/ws';
  const { container } = render(<App />)
  const element = screen.getAllByText('Connect')
  expect(element).toHaveLength(2)
  expect(element[0]).toBeInTheDocument()
  expect(element[1]).toBeInTheDocument()
  expect(container).toMatchSnapshot();
});
