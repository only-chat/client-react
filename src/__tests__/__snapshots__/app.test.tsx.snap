// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App 1`] = `
<div>
  <h1>
    Connect
  </h1>
  <div>
    <label
      for="username"
    >
      Host:
    </label>
    <input
      id="username"
      value="http://github.com/only-chat/ws"
    />
  </div>
  <button>
    Connect
  </button>
</div>
`;
